src/form/Form.jsx

```diff
@@ -79,8 +80,10 @@ export default class Form extends React.Component {
     validateWaitMs: PropTypes.number,
     normalize: PropTypes.func,
     disabled: PropTypes.bool,
-    children: PropTypes.oneOfType([PropTypes.node, PropTypes.func]).isRequired
-  }
+    children: PropTypes.oneOfType([PropTypes.node, PropTypes.func]).isRequired,
+    serverValidationErrors: PropTypes.any,
```

Comment:
It is not `any`. Please specify a precise `PropTypes.Shape`.

Reply:
we also have `any` in `values`, should I replace this too?

Reply:
as you wish

Reply:
btw, `PropTypes.shape` not always fine for some reason (it throwing warnings), so I used `PropTypes.object`(not sure if its fine though)